todoman/cli.py

```diff
@@ -148,6 +149,10 @@ def new(ctx, summary, list, todo_properties, interactive):
     if not todo.summary:
         raise click.UsageError('No SUMMARY specified')
 
+    default_due = int(ctx.obj['config']['main'].get('default_due', '24'))
```

Comment:
Looking at that trace, I'm not sure _what_ isn't working. If you set it to `int(default=24)`, it should never be `None`, but you also don't need to re-cast here.